src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not typically necessary for entity classes. Entities are usually not meant to be cloned as they represent database records, and cloning them can lead to unintended side effects, such as duplicate records or confusion over identity in the context of a persistence framework like Hibernate.